packages/client/README

```diff
@@ -0,0 +1,72 @@
+# Running Tests
+
+This project includes unit tests and end-to-end (E2E) tests. Below are the instructions to run the E2E tests.
+
+## Steps to Run E2E Tests
+
+1. In the terminal, navigate to the contracts directory:
+````cmd
+    cd packages/contracts && forge build
+````
+
+2. After building the contracts, return to the root directory:
+````cmd
+    cd .. && cd ..
+````
+
+3. Start the local Hardhat node:
+````cmd
+    yarn hardhat node
+````
+
+4. With the local Hardhat node running, execute the E2E tests:
+````cmd
+    yarn client hardhat test ./test/l2_e2e.spec.ts --network localhost
```

Comment:
this could be a script on the client's package.json as well.

perhaps even following [this example](https://github.com/blockful-io/external-resolver/blob/main/packages/contracts/package.json#L7)